### PR TITLE
Create eo.yml

### PR DESCRIPTION
**Pull request by 1000283**
Esperanto translation

### CODE REVIEW

File: i18n/eo.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+- id: prev_page
+  translation: "« paĝo"
```
Comment (adityatelange) [inline i18n/eo.yml:2]: Can you remove these arrows? The template adds arrows, we don't need to add them in translation files

File: i18n/eo.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+- id: next_page
+  translation: "paĝo »"
```
Comment (adityatelange) [inline i18n/eo.yml:5]: Here too

File: i18n/eo.yml (hunk 1 above)
Comment (1000283) [reply i18n/eo.yml]: Done

**Review by adityatelange — APPROVED** [review]
Thanks :+1:

**Pull request merged**